conf/nginx/viahtml/uwsgi_proxy.conf

```diff
@@ -14,11 +14,4 @@ uwsgi_busy_buffers_size    256k;
 
 uwsgi_pass rewriter;
 
-add_header "X-Via" "html";
-
-# THIS IS ALMOST CERTAINLY WRONG... we should find a way to pass
-# through the Cache-Control header from the original source rather
-# than implying everything is safe to cache. It won't be
-# private - Only the browser should cache, not Cloudflare etc
-proxy_hide_header "Cache-Control";
-add_header "Cache-Control" "private, max-age=300, stale-while-revalidate=300";
```

Comment:
This has to go away, and really should have in the previous `Cache-Control` PR. Otherwise NGINX adds this header in addition to the header emitted by the app.

If the two headers conflict the interpretation is likely undefined.